.github/workflows/build-cosi.yaml

```diff
@@ -0,0 +1,55 @@
+---
+name: COSI Builds
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'  # Every sunday night midnight
+  pull_request:
```

Comment:
Add a path specifier
```
    pull_request:
      path:
        - "cose-*"
        - ".github/workflows/build-cosi.yaml"
```
So this only runs when modifications to the cosi driver are made. Don't need to build the cosi driver when just the docs change.